### ISSUE TITLE
Use - Crash when brushes from disabled bundles are referenced

### ISSUE THREAD

**Issue by Grum999** [Grum999/BuliBrushSwitch#1]
From Krita Artist topic: https://krita-artists.org/t/buli-brush-switch/32677/80

>An error occurred today after I disabled some bundles. Every time krita starts, it will jump out several times.
>
> I only put two brushes in it. I guess it’s the bug caused by the disappearance of one brush. After recovery, the plug-in will no longer report errors.
>
> However, it is troublesome to modify the plug-in every time you operate the bundle…

Need to reproduce case for a deeper analysis as normally if a brush doesn't exists there's a fallback 
Maybe, as the brush still exist (but is in a disabled bundle) the fallback is not working properly

Anyway, whatever the origin of the problem is, unhandled Python exception is not a proper thing 

```
TypeError
Python 3.8.1: C:\GAMES\STEAM\steamapps\common\Krita\krita\bin\krita.exe
Tue May  3 12:08:49 2022

A problem occurred in a Python script.  Here is the sequence of
function calls leading up to the error, in the order they occurred.

 C:\Users\123\AppData\Roaming\krita\pykrita\bulibrushswitch\bbs\bbswbrushes.py in data(self=<bulibrushswitch.bbs.bbswbrushes.BBSBrushesModel object>, index=<PyQt5.QtCore.QModelIndex object>, role=1)
  946                 if column==BBSBrushesModel.COLNUM_ICON:
  947                     # QIcon
  948                     return QIcon(QPixmap.fromImage(item.image()))
  949         elif role == Qt.ToolTipRole:
  950             id=self.__items[row]
global QIcon = <class 'PyQt5.QtGui.QIcon'>
global QPixmap = <class 'PyQt5.QtGui.QPixmap'>
QPixmap.fromImage = <built-in function fromImage>
item = <BBSBrush(4487ff9d-99b7-44ba-b244-23f9c6b58275, @ A Base color, #ffffff)>
item.image = <bound method BBSBrush.image of <BBSBrush(4487ff...4ba-b244-23f9c6b58275, @ A Base color, #ffffff)>>
TypeError: fromImage(QImage, flags: Union[Qt.ImageConversionFlags, Qt.ImageConversionFlag] = Qt.AutoColor): argument 1 has unexpected type 'NoneType'
    __cause__ = None
    __class__ = <class 'TypeError'>
    __context__ = None
    __delattr__ = <method-wrapper '__delattr__' of TypeError object>
    __dict__ = {}
    __dir__ = <built-in method __dir__ of TypeError object>
    __doc__ = 'Inappropriate argument type.'
    __eq__ = <method-wrapper '__eq__' of TypeError object>
    __format__ = <built-in method __format__ of TypeError object>
    __ge__ = <method-wrapper '__ge__' of TypeError object>
    __getattribute__ = <method-wrapper '__getattribute__' of TypeError object>
    __gt__ = <method-wrapper '__gt__' of TypeError object>
    __hash__ = <method-wrapper '__hash__' of TypeError object>
    __init__ = <method-wrapper '__init__' of TypeError object>
    __init_subclass__ = <built-in method __init_subclass__ of type object>
    __le__ = <method-wrapper '__le__' of TypeError object>
    __lt__ = <method-wrapper '__lt__' of TypeError object>
    __ne__ = <method-wrapper '__ne__' of TypeError object>
    __new__ = <built-in method __new__ of type object>
    __reduce__ = <built-in method __reduce__ of TypeError object>
    __reduce_ex__ = <built-in method __reduce_ex__ of TypeError object>
    __repr__ = <method-wrapper '__repr__' of TypeError object>
    __setattr__ = <method-wrapper '__setattr__' of TypeError object>
    __setstate__ = <built-in method __setstate__ of TypeError object>
    __sizeof__ = <built-in method __sizeof__ of TypeError object>
    __str__ = <method-wrapper '__str__' of TypeError object>
    __subclasshook__ = <built-in method __subclasshook__ of type object>
    __suppress_context__ = False
    __traceback__ = <traceback object>
    args = ("fromImage(QImage, flags: Union[Qt.ImageConversio...Color): argument 1 has unexpected type 'NoneType'",)
    with_traceback = <built-in method with_traceback of TypeError object>

The above is a description of an error in a Python program.  Here is
the original traceback:

Traceback (most recent call last):
  File "C:\Users\123\AppData\Roaming\krita\pykrita\bulibrushswitch\bbs\bbswbrushes.py", line 948, in data
    return QIcon(QPixmap.fromImage(item.image()))
TypeError: fromImage(QImage, flags: Union[Qt.ImageConversionFlags, Qt.ImageConversionFlag] = Qt.AutoColor): argument 1 has unexpected type 'NoneType'
```

```
AttributeError
Python 3.8.1: C:\GAMES\STEAM\steamapps\common\Krita\krita\bin\krita.exe
Tue May  3 12:09:29 2022

A problem occurred in a Python script.  Here is the sequence of
function calls leading up to the error, in the order they occurred.

 C:\Users\123\AppData\Roaming\krita\pykrita\bulibrushswitch\bbs\bbswbrushes.py in paint(self=<bulibrushswitch.bbs.bbswbrushes.BBSBrushesModelDelegate object>, painter=<PyQt5.QtGui.QPainter object>, option=<PyQt5.QtWidgets.QStyleOptionViewItem object>, index=<PyQt5.QtCore.QModelIndex object>)
 1356                 painter.fillRect(option.rect, option.palette.color(QPalette.Highlight))
 1357 
 1358             painter.drawPixmap(option.rect.topLeft(), index.data(Qt.DecorationRole).pixmap(option.decorationSize))
 1359             painter.restore()
 1360             return
painter = <PyQt5.QtGui.QPainter object>
painter.drawPixmap = <built-in method drawPixmap of QPainter object>
option = <PyQt5.QtWidgets.QStyleOptionViewItem object>
option.rect = PyQt5.QtCore.QRect(0, 99, 32, 394)
option.rect.topLeft = <built-in method topLeft of QRect object>
index = <PyQt5.QtCore.QModelIndex object>
index.data = <built-in method data of QModelIndex object>
global Qt = <class 'PyQt5.QtCore.Qt'>
Qt.DecorationRole = 1
).pixmap undefined
option.decorationSize = PyQt5.QtCore.QSize(32, 32)
AttributeError: 'NoneType' object has no attribute 'pixmap'
    __cause__ = None
    __class__ = <class 'AttributeError'>
    __context__ = None
    __delattr__ = <method-wrapper '__delattr__' of AttributeError object>
    __dict__ = {}
    __dir__ = <built-in method __dir__ of AttributeError object>
    __doc__ = 'Attribute not found.'
    __eq__ = <method-wrapper '__eq__' of AttributeError object>
    __format__ = <built-in method __format__ of AttributeError object>
    __ge__ = <method-wrapper '__ge__' of AttributeError object>
    __getattribute__ = <method-wrapper '__getattribute__' of AttributeError object>
    __gt__ = <method-wrapper '__gt__' of AttributeError object>
    __hash__ = <method-wrapper '__hash__' of AttributeError object>
    __init__ = <method-wrapper '__init__' of AttributeError object>
    __init_subclass__ = <built-in method __init_subclass__ of type object>
    __le__ = <method-wrapper '__le__' of AttributeError object>
    __lt__ = <method-wrapper '__lt__' of AttributeError object>
    __ne__ = <method-wrapper '__ne__' of AttributeError object>
    __new__ = <built-in method __new__ of type object>
    __reduce__ = <built-in method __reduce__ of AttributeError object>
    __reduce_ex__ = <built-in method __reduce_ex__ of AttributeError object>
    __repr__ = <method-wrapper '__repr__' of AttributeError object>
    __setattr__ = <method-wrapper '__setattr__' of AttributeError object>
    __setstate__ = <built-in method __setstate__ of AttributeError object>
    __sizeof__ = <built-in method __sizeof__ of AttributeError object>
    __str__ = <method-wrapper '__str__' of AttributeError object>
    __subclasshook__ = <built-in method __subclasshook__ of type object>
    __suppress_context__ = False
    __traceback__ = <traceback object>
    args = ("'NoneType' object has no attribute 'pixmap'",)
    with_traceback = <built-in method with_traceback of AttributeError object>

The above is a description of an error in a Python program.  Here is
the original traceback:

Traceback (most recent call last):
  File "C:\Users\123\AppData\Roaming\krita\pykrita\bulibrushswitch\bbs\bbswbrushes.py", line 1358, in paint
    painter.drawPixmap(option.rect.topLeft(), index.data(Qt.DecorationRole).pixmap(option.decorationSize))
AttributeError: 'NoneType' object has no attribute 'pixmap'
```

**Comment by Grum999** [comment]
When a brush is linked to a deactivated bundle, brush thumbnail can't be returned and then generate python exception

To avoid this, return a warning icon:
<img width="469" alt="image" src="https://user-images.githubusercontent.com/63422799/194847598-e246022d-f9b8-43bd-b859-164794685430.png">

**Comment by Grum999** [comment]
Will be closed with release v1.0.0
